openedx/openedx-platform
# build: expose working openedx/lms and openedx/cms docker images

## Description

This PR prepares edx-platform's experimental Dockerfile for optional use in devstack. Presently, the image built by this Dockerfile isn't used anywhere.

Notable changes:
* Drop the `openedx/edx-platform` image name in favor of `openedx/lms` and `openedx/cms`.
* Drop the newrelic stages and tags.
* Create `openedx/lms-dev` and `openedx/cms-dev` image variants which use Django devserver, install dev requirements, and specify devstack Django settings.
* Add `lms.yml` and `cms.yml` configuration files, extracted from the existing edxapp docker image.
* Adds three new scripts, each of which replaces an Ansible or Paver-supported function with a pure bash + Django management command implementation.

## Testing instructions

See the testing instructions for the [devstack PR](https://github.com/edx/devstack/pull/866).

## Other information

~This is blocked on [the CMS vs Studio ADR](https://github.com/edx/edx-platform/pull/29534). Whatever the result of that ADR is, I'll update the image tags here to match.~ Resolved.

## Supporting information

This is a dependency attempt to [use the experimental images in devstack](https://github.com/edx/devstack/pull/866), which itself is part of the effort to [move off of edxops DockerHub org images](https://github.com/edx/devstack/issues/869).
